### PR TITLE
Update code owner file

### PR DESCRIPTION
**Pull request by SherlockNoMad**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+orttraining/orttraining/python/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+orttraining/orttraining/test/python/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+orttraining/pytorch_frontend_examples/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/python/training/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_ort_trainer.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_ort_trainer_with_mixed_precision.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_training_unit_tests.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_training_unittest_utils.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+samples/python/**@thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
```
Comment (thiagocrepaldi) [inline CODEOWNERS:13]:
```suggestion
samples/python/**@thiagocrepaldi @BowenBao @liqunfu @baijumeswani @SherlockNoMad @tlh20
```

Comment (thiagocrepaldi) [reply CODEOWNERS:13]: there is some dupe here

Comment (jywu-msft) [reply CODEOWNERS:13]: please remove the ownership on samples/python/** line or carve out a specific training subdirectory.

Comment (thiagocrepaldi) [reply CODEOWNERS:13]: @SherlockNoMad you could create an ormodule folder inside samples/python to separate from the other components, such as inference or quantization

**Review by natke — COMMENTED** [review]
Some duplication with #8335 8310

**Review by thiagocrepaldi — DISMISSED** [review]

**Review by baijumeswani — APPROVED** [review]

File: orttraining/orttraining/test/python/_test_commons.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def generate_dummy_optim_state(model, optimizer):
 
 def _load_pytorch_transformer_model(device, dynamic_axes=False, legacy_api=False, data_dir=None):
     # Loads external Pytorch TransformerModel into utils
-    pytorch_transformer_path = os.path.join('samples', 'python', 'pytorch_transformer')
+    pytorch_transformer_path = os.path.join('samples', 'python', 'training', 'orttrainer', 'pytorch_transformer')
```
Comment (thiagocrepaldi) [inline orttraining/orttraining/test/python/_test_commons.py:144]:
```suggestion
    pytorch_transformer_path = os.path.join('samples', 'python', 'training', 'orttrainer')
```

**Review by thiagocrepaldi — APPROVED** [review]

**Pull request merged**